KubeArmor/.goreleaser.yaml

```diff
@@ -58,22 +62,18 @@ nfpms:
         type: symlink
       - src: ./karmor
         dst: /usr/local/bin/karmor
-      - src: ./BPF/*
-        dst: /opt/kubearmor/BPF/
-      - src: ./templates/*
-        dst: /opt/kubearmor/templates/
     scripts:
       postinstall: packaging/post-install.sh
     overrides:
       deb:
         dependencies:
-          - make
-          - libelf-dev
+          - clang
+          - llvm
           - linux-headers-generic
       rpm:
         dependencies:
-          - make
-          - elfutils-libelf-devel
```

Comment:
Do we not need these?
Not sure but ig we need libelf for compiling the monitor code 🤔

Reply:
@nam-jaehyun , can you respond to this comment? Lets get this one merged so that we can create a new systemd release. Thanks

Reply:
When I tested the rpm package, KubeArmor worked without those dependencies. However, I also had the same question. We may see if those are required later.

Reply:
It might be possible that these exist already on the system? (which is most likely the case). If the dependencies already exist it wouldn't reinstall so should be fine to add them 🤔

Reply:
added make and libelf-dev (elfutils-libelf-devel)